_stbt/ocr.py

```diff
@@ -363,6 +391,57 @@ def match_text(text, frame=None, region=Region.ALL,
     return result
 
 
+# Python 2.7 & 3.6 have `re._pattern_type` but that will be removed in Python
+# 3.7 where they introduce `re.Pattern`.
```

Comment:
I'm bemused by Python's attitude towards backward compatibility.  How high can the maintenance burden of adding `_pattern_type = Pattern` be?